test/tests/tsdx-build-closure-compiler.test.js

```diff
@@ -0,0 +1,90 @@
+/**
+ * @jest-environment node
+ */
+
+const shell = require('shelljs');
+const util = require('../fixtures/util');
+
+shell.config.silent = false;
+
+const grep = (pattern, file) => {
+  const output = shell.grep(pattern, file);
+  const grepOutput = output.stdout.replace(`\n`, ``);
+  return grepOutput.length > 0;
+};
+
+const stageName = 'stage-build-closure-compiler';
+
+describe('tsdx build with closure compiler default options', () => {
+  beforeAll(() => {
+    util.teardownStage(stageName);
+  });
+
+  it('should minify bundle with default options', () => {
+    util.setupStageWithFixture(stageName, 'build-withConfig');
+
+    let output = shell.exec(
+      'node ../dist/index.js build --env production --closureCompiler'
```

Comment:
`--env production` doesn't actually do anything (and isn't an official flag for build either). See also #508

Reply:
fixed